client/src/components/App.jsx

```diff
@@ -20,8 +20,8 @@ function App() {
   const [checkingDatesSet, setCheckingDatesSet] = useState(false);
   const [focusedDate, setFocusedDate] = useState(undefined);
   function getListingData() {
-    const listingID = window.location.pathname.split('/')[1];
-    return axios.get(`/api/listings/${isNaN(listingID) ? 0 : listingID}`)
+    const listingID = window.location.pathname.split('/')[2];
+    return axios.get(`/api/listings/${listingID}`)
```

Comment:
are you planning on only using a url to the proxy server with an attached id?

I just inserted a couple of lines here to handle a default case if you want to use!

`actualID = Number(listingID) || 21
  return axios.get(`api/listings/${actualID}`)`

Reply:
updated this for next commit